guides/wallet.html

```diff
@@ -34,7 +34,8 @@
     <li><a id="guides-claims"   href="/guides/claims.html">Name Claims</a></li>
     <li><a id="guides-auctions"   href="/guides/auctions.html">Name Auctions</a></li>
     <li><a id="guides-wallet"  class="active"  href="/guides/wallet.html">Wallet Setup</a></li>
-    <li><a id="guides-protocol"   href="/guides/protocol.html">Protocol Summary</a></li>
+    <li><a id="guides-resource-records"   href="/guides/resource-records.html">Handshake Resource Records</a></li>
+    <li><a id="guides-protocol"  class="active"  href="/guides/protocol.html">Protocol Summary</a></li>
```

Comment:
this is the other side of that rebase-on-master thing

(the wrong title is highlighted in the sidebar)